siliconcompiler/core.py

```diff
@@ -675,6 +675,16 @@ def load_checklist(self, name):
         else:
             self.error(f'Checklist module {name} not found in $SCPATH or siliconcompiler/checklists/.')
 
+    ##########################################################################
+    def use(self, module):
+        '''
+        Loads a SiliconCompiler module into the current chip object by calling
+        a module.setup() method.
+        '''
+
+        self.import_library(module.setup())
```

Comment:
Yes, we should discuss. Moving to one chip object/module seems tedious, it would require creating 10s of files of boilerplate for more advanced libraries.

Reply:
Yes, but it's done once per library...considering that downloading and procuring libraries for advanced nodes is a huge investment (and involves terabytes of data) I am not sure the boiler plate effort of putting together a python module per lib is big enough? It still seems like a corner case that should not drive the architecture?

Reply:
...on the other hand, reconsidering. Besides the library, there are instances where we have a large number of verilog files where by convention people put one module per file. There is no way we would want to have a python wrapper for each module. Returning a list of objects would be pretty nice for a library of open source verilog components, each chip object possible representing a trivial module (nand, flip flop)....